src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.css

```diff
@@ -1,20 +1,26 @@
 .header {
-    display: flex;
-    flex-wrap: wrap;
-    font-weight: bold;
+  display: flex;
+  flex-wrap: wrap;
+  font-weight: bold;
 }
 .label {
-    font-weight: bold;
-    margin-bottom: 0px;
+  font-weight: bold;
+  margin-bottom: 0px;
 }
 ul {
-    margin: 5px 0px;
-    padding: 0px;
+  margin: 5px 0px;
+  padding: 0px;
 }
 li {
-    display: flex;
-    flex-wrap: wrap;
-    list-style-type: none;
-    border: 2px solid #E1BEBD;
-    padding: 10px 0px;
-}
\ No newline at end of file
+  display: flex;
+  flex-wrap: wrap;
+  list-style-type: none;
+  border: 2px solid #e1bebd;
```

Comment:
I don't think this color works well with this screen. @marieanik @KDNeufeld

Reply:
The colour was pulled straight out of the wireframe.  @marieanik, let me know if there's an update to the design I can incorporate. Thanks!

Reply:
Done. Used colour from a different part of the wireframe.